docs(sessions): :memo: revise and simplify stats text (#233)
rostools/r-cubed-advanced: sessions/stats-analyses-basic.qmd

More specific objectives are to:

1.  Describe the general "workflow": State the research question,
    construct a model, prepare the data, fit the model, and extract the
    results.
2.  Categorize the model definition step as a distinct, theory-driven
    step, separate from the data, and use `{parsnip}` functions to help
    with defining your model.

> 3.  Identify data transformation techniques and evaluate which are good

Author: ```suggestion
3.  Identify some data transformation techniques and evaluate which are good
```

    options given the data. Use functions in the `{recipes}` package to
    apply these transformations to your data.
4.  Use the `{broom}` package to extract the model results to later
    present them in graphical format (with `{ggplot2}`).
5.  Continue applying the concepts and functions used from the previous
    sessions.

> Specific "anti"-objectives:
> 

Author: ```suggestion
Specific "anti"-objectives that we will *not* learn:
```

> We will *not* learn:

Author: ```suggestion
```

> -   How to choose and apply the appropriate statistical model or tests.
> 
> -   Statistical theory.
> 
> -   How to determine relevant data transformations for statistical
>     tests.
> 
> -   How to interpret the statistical results.

Author: ```suggestion
-   How to choose and apply the appropriate statistical model or tests.
-   Statistical theory.
-   How to determine relevant data transformations for statistical
    tests.
-   How to interpret the statistical results.
```

Just to have it more compact when displayed on the website.

<!-- TODO: Create a revealjs presentation on this? -->

> :::::::: callout-note

Author: ```suggestion
:::: callout-note
```

I think this was automatically added because of the multiple callouts below, so might not be auto added again.

## Reading task: \~15 minutes

> 

Author: ```suggestion

Research, especially quantitative research, is about making inferences 
about the world by quantifying uncertainty from some data.
And we quantify uncertainty by using statistics and statistical models.

```

> A statistical model is a simple way to describe the real world using
> mathematics. In research, we often aim to understand the relationships
> between multiple variables. For example, if we change the predictor
> (independent variable) ( x ), how does that affect the outcome
> (dependent variable) ( y )?

Author: ```suggestion
A statistical model is a simple way to describe the real world using
mathematics. In research, we often aim to understand the relationships
between multiple variables. For example, if we change the predictor
(independent variable or $x$), how does that affect the outcome
(dependent variable or $y$)?
```

Some simple examples are:

> -   "How does this new cancer treatment affect survival rates?"
> 
> -   "How does this new user interface affect user watch time?"

Author: ```suggestion
-   "How does this new cancer treatment affect survival rates?"
-   "How does this new user interface affect user watch time?"
```

Nice question that isn't related to health at all, I like :grin:

or more complex:

> -   "How does sunlight and water affect plant growth?"
> 
> -   "What is the relationship between a metabolite and T1D compared to
>     controls, adjusting for age and gender?"

Author: ```suggestion
-   "How does sunlight and water affect plant growth?"
-   "What is the relationship between a metabolite and type 1 diabetes compared to
    controls, adjusting for age and gender?"
```

We try to reduce the use of abbreviations, so I wrote out the full T1D here.

These relationships can involve single or multiple predictors (such as
sunlight and water, or metabolite, gender, and age). The outcomes can be
either continuous (such as plant growth) or categorical (such as whether

> a person has T1D or not).

Author: ```suggestion
a person has type 1 diabetes or not).
```

The relationship between predictors and the outcome is known as a

> regression. If there is a relationship between ( x ) and ( y ), the

Author: ```suggestion
regression. If there is a relationship between ($x$) and ($y$), the
```

> corresponding mathematical object is a function:

Author: ```suggestion
corresponding mathematical object is a function, $f()$:
```

> ::: callout-note
> $$y = f(x) $$
> :::

Author: ```suggestion
$$y = f(x)$$
```

Callout blocks for this situation for equations isn't necessary. We could add an equation number beside though?

Since no relationship is perfect, we add some random error (error) to
describe the difference between the function and the actual measured
variables:

> ::: callout-note
> $$y = f(x) + error $$
> :::

Author: ```suggestion
$$y = f(x) + error$$
```

The simplest form of regression is linear regression, where the formula
is a linear combination:

> :::: callout-note
> ::: callout-note
> $$y = intercept + x + e$$
> :::
> ::::

Author: ```suggestion
$$y = intercept + x + error$$
```

> Here, the intercept is the value of ( y ) when ( x ) is zero.

Author: ```suggestion
Here, the $intercept$ is the value of $y$ when $x$ is zero.
```

    Sunlight --- Water
```

or mathematically:

$$growth = sunlight + water$$

This expression is known as our theoretical model. However, not all
theoretical models can be tested against the real world. We need to

> structure the theoretical model so that we can measure the parameters

Author: ```suggestion
structure the theoretical model so that we can measure the parameters---the variables like growth, sunlight, and water in the model above---
```

involved. In this case, we need to measure plant growth (e.g., weight in
grams), the amount of water given per day (in liters), and the number of
sunlight hours per day.

This also means that when formulating a research question, there is an
order, or workflow, to follow:

> ### Workflow for Statistical Models
> 

Author: ```suggestion
```

Since the next list is connected directly to the paragraph above, inserting a header here would break that connection, so I removed it :relaxed:

    model_est -- "plot_estimates()" --> plot_est[Plot of estimates]
    plot_est -- "tar_read()" --> qmd[Quarto]
```

> ::::::::

Author: ```suggestion
::::
```

choice is logistic regression, which requires a binary outcome variable.

> So we have the theoretical model and the type of model to use - how do

Author: ```suggestion
So we have the theoretical model and the type of model to use, now how do
```

we express this as code in R? There are many ways of doing the same
thing in R, but some are a bit easier than others.

One approach that is quite generic and fits with the ideals of the
`{tidyverse}`, is a similar universe of packages called the
`{tidymodels}`. We teach `{tidymodels}`because they are created by
experienced software developers at Posit, the same company behind the
`{tidyverse}` and RStudio. They are known for excellent documentation.

> `{tidymodels}` make it easy to create and use models in a flexible way,

Author: ```suggestion
`{tidymodels}` makes it easy to create and use models in a flexible way,
```

allowing you to switch model types or data processing methods without
needing to learn new tools. This makes your work more efficient and
adaptable.

> `{tidymodels}` are designed to clearly separate the different parts of

Author: ```suggestion
`{tidymodels}` is designed to clearly separate the different parts of
```
